docs/Examples/BbhInitialData/BbhInitialData.ipynb

```diff
@@ -0,0 +1,463 @@
+      "Gmres(3) iteration complete. Remaining residual: 5.288583e-06\n",
+      "Gmres(4) iteration complete. Remaining residual: 4.922907e-06\n",
+      "Gmres(5) iteration complete. Remaining residual: 4.250901e-06\n",
+      "Gmres(6) iteration complete. Remaining residual: 1.596286e-06\n",
+      "Gmres has converged in 6 iterations: RelativeResidual - The residual magnitude has decreased to a fraction of 0.001 of its initial value or below (0.00094429).\n",
+      "NewtonRaphson(4) iteration complete (0 globalization steps, step length 1). Remaining residual: 1.606242e-06\n",
+      "Gmres initialized with residual: 1.606242e-06\n",
+      "Gmres(1) iteration complete. Remaining residual: 7.008839e-07\n",
+      "Gmres(2) iteration complete. Remaining residual: 5.285042e-07\n",
+      "Gmres(3) iteration complete. Remaining residual: 4.919028e-07\n",
+      "Gmres(4) iteration complete. Remaining residual: 3.871318e-07\n",
+      "Gmres(5) iteration complete. Remaining residual: 1.603757e-07\n",
+      "Gmres(6) iteration complete. Remaining residual: 2.922369e-08\n",
+      "Gmres(7) iteration complete. Remaining residual: 1.438336e-08\n",
+      "Gmres(8) iteration complete. Remaining residual: 5.271254e-09\n",
+      "Gmres(9) iteration complete. Remaining residual: 2.125873e-09\n",
+      "Gmres(10) iteration complete. Remaining residual: 1.044162e-09\n",
+      "Gmres has converged in 10 iterations: RelativeResidual - The residual magnitude has decreased to a fraction of 0.001 of its initial value or below (0.000650065).\n",
+      "NewtonRaphson(5) iteration complete (0 globalization steps, step length 1). Remaining residual: 1.066549e-09\n",
+      "Gmres initialized with residual: 1.066549e-09\n",
+      "Gmres(1) iteration complete. Remaining residual: 7.999901e-10\n",
+      "Gmres(2) iteration complete. Remaining residual: 5.991592e-10\n",
+      "Gmres(3) iteration complete. Remaining residual: 4.073678e-10\n",
+      "Gmres(4) iteration complete. Remaining residual: 2.559960e-10\n",
+      "Gmres(5) iteration complete. Remaining residual: 1.242241e-10\n",
+      "Gmres(6) iteration complete. Remaining residual: 3.521827e-11\n",
+      "Gmres has converged in 6 iterations: AbsoluteResidual - The residual magnitude has decreased to 1e-10 or below (3.52183e-11).\n",
+      "NewtonRaphson(6) iteration complete (0 globalization steps, step length 1). Remaining residual: 3.585636e-11\n",
+      "NewtonRaphson has converged in 6 iterations: AbsoluteResidual - The residual magnitude has decreased to 1e-10 or below (3.58564e-11).\n",
+      "\n",
+      "Done!\n",
+      "Wall time in seconds: 180.425137\n",
+      "Date and time at completion: Sun Feb 20 00:37:23 2022\n",
+      "\n",
+      "[Partition 0][Node 0] End of program\n"
+     ]
+    }
+   ],
+   "source": [
+    "NUM_CORES = 15\n",
+    "!source {os.path.join(SPECTRE_HOME, 'support/Environments/minerva_gcc.sh')} \\\n",
```

Comment:
I found this pretty difficult to parse and I would consider myself reasonably well-versed in bash+python. I think if the executable is run inside the container this'll be easier to understand (no module loading, etc.), and maybe define an env var for `SOLVE_XCTS_EXECUTABLE`? Then in the markdown above this explain a bit the arguments? Eg `We named the yaml input file Bbh.yaml, and run on NUM_CORES` Maybe make the input file name an argument at the top after `SPECTRE_HOME`?